openmetadata-ui/src/main/resources/ui/src/utils/CommonUtils.tsx

```diff
@@ -1073,3 +1073,7 @@ export const getLoadingStatus = (
     displayText
   );
 };
+
+// return array of id's
+export const generateIdArray = (arr: EntityReference[] | undefined) =>
+  arr?.map((item) => item.id);
```

Comment:
What will be the return type if arr will be undefined?

Reply:
@chirag-madlani  It will be undefined only.